elastic/kibana
# [Entity Analytics] [Entity Store] Refactor entity store enablement (server side)

## Summary

This PR adds 2 new endpoints regarding enablement of the Entity Store:
  * `api/entity_store/enable`, which initializes entity engines for both `user` and `host` entities
  * `api/entity_store/status`, which computes a global store status based on the individual engine status

In addition, running initialization of multiple engines in parallel is now allowed.


### How to test

1. Use dev tools to call `POST kbn:/api/entity_store/enable`
2. Check that two engines were created and that the status is `installing` by calling `GET kbn:/api/entity_store/status`
3. Wait a few seconds and keep calling the `status` endpoint. Once initialization finishes, the status should switch to `running`
